ahcer/src/app/episode-report/episode-report.component.html

```diff
@@ -0,0 +1,125 @@
+<div class="component-container">
+  <h2 class="header" *ngIf="!loadingPatient && currentPatient">
+    Complete episode list
+    <span class="not-printable"> for {{currentPatient.firstName}} {{currentPatient.lastName}}</span>
+  </h2>
+  <p *ngIf="!loadingPatient && !currentPatient">No patients found. Please add one.</p>
+
+  <mat-form-field appearance="fill" class="not-printable" *ngIf="patients && patients.length > 1">
+    <mat-label>Choose a different patient</mat-label>
+    <mat-select (selectionChange)="changePatient($event.value)">
+      <mat-option *ngFor="let patient of patients" [value]="patient.id">{{patient.firstName}}</mat-option>
+    </mat-select>
+  </mat-form-field>
+  <div class="table-container mat-elevation-z8" *ngIf="episodes_count > 0" >
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="startTime">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Start Date </th>
+        <td mat-cell *matCellDef="let element"> {{element.startTime?.toDate()| date: 'MM/dd/yyyy'}} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="endTime">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> End Date </th>
+        <td mat-cell *matCellDef="let element"> {{element.endTime?.toDate()| date: 'MM/dd/yyyy'}} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="duration">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header>Duration</th>
+        <td mat-cell *matCellDef="let result">{{calculateDuration(result.startTime, result.endTime)}}</td>
```

Comment:
I think we could go with just hours instead of days and hours (2 days 1 hour) = 49h

Reply:
The thing about this is that we used days, hrs, mins, and secs for the duration in episode listing (at /episodes). Do you think I should change that part to D, h, m, and s too? Or should change it so it uses just h when the duration is higher than an hour?

Reply:
If it uses the same method then I would make it consistent in both locations. If it isn't then just change the report. We could also just add this as another story and address later.